integration-intygstjanst/src/main/java/se/inera/intyg/minaintyg/integration/intygstjanst/client/SendCertificateUsingIntygstjanstService.java

```diff
@@ -0,0 +1,48 @@
+package se.inera.intyg.minaintyg.integration.intygstjanst.client;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+import se.inera.intyg.minaintyg.integration.api.certificate.SendCertificateIntegrationRequest;
+
+@Service
+public class SendCertificateUsingIntygstjanstService {
```

Comment:
Realised that we are missing a test-file for this class. It would probably have catched the  error with the url. Would be great to have a test class to verify the details here.

Reply:
Tried adding a test but I'm a bit unsure of what to test so let me know if I forgot something